.gitignore

```diff
@@ -11,6 +11,7 @@ subdir-static
 test-storybook
 chromatic-build-*.xml
 chromatic-diagnostics.json
-# the dist folder IS included so we can test every version in the CI without publishing
```

Comment:
This was [added](https://github.com/chromaui/chromatic-cli/commit/229ca6c919dd4856850f46c3caa7d3fc0f22c1af) a long time ago. I don't know how relevant it is. I suspect this is intended to prevent `dist` from being added to the repo by a CI script that builds the CLI for testing. I'm not sure whether we have such a script anymore. Can't hurt to keep ignoring `dist` but the comment can go 🤷